conf/messages

```diff
@@ -186,43 +186,47 @@ pdf.ruling.section.ruling.legalInformation.section2.paragraph2.listitem1 = infor
 pdf.ruling.created.label = File created
 
 
-pdf.application.title = Application for a goods classification ruling
-pdf.application.submitted = Application submitted on
-pdf.application.reference = Application reference number
-pdf.application.section.applicant.heading = Applicant details
+pdf.application.title = Your application for a Binding Tariff Information ruling
```

Comment:
Similar - I assume you copied all of the "pdf.application...." messages from one project to the other

Reply:
Why not?? I think we should move it anyway... Not big fan to put into the Back End, but at least would be just one implementation.

Reply:
I wasn't suggesting that was wrong.   Just checking that the content (text) was the same in both places